src/views/LimitedRestorationExtension.vue

```diff
@@ -271,6 +262,19 @@ export default class LimitedRestorationExtension extends Vue {
       // set applicant orgPerson
       entitySnapshot.orgPersons = this.parseApplicantOrgPerson(applicant)
 
+      this.setEntitySnapshot(entitySnapshot)
+
+      if (!restorationFiling.restoration.expiry) {
+        // New limited restoration extension
+        // Set the previously filed limited restoration in the store.
+        await this.setStateFilingRestoration()
+        // parse draft restoration filing into store
+        this.parseRestorationFiling(restorationFiling)
+      } else {
+        this.parseRestorationFiling(restorationFiling)
+        await this.setStateFilingRestoration()
+      }
+
       if (!this.restorationResource) {
         throw new Error(`Invalid restoration resource entity type = ${this.getEntityType}`)
       }
```

Comment:
Is the code on line 293-294 still needed, since you deleted the notice?

Check what the other filings do... Do they all set the fee prices from filing data, or do some skip that? I'm wondering if it needs to be there for an external reason or not for this filing.

Reply:
Ditto for LimitedRestorationToFull.vue lines 312-313.

Reply:
@severinbeauvais ,other filing set this from Filing data. I will test a bit more after removing this.

![image](https://user-images.githubusercontent.com/116035339/229883539-a4b54afe-426f-4b5a-b92b-ddf1b1a76bee.png)

Reply:
Yes, please test a bit. (I see corrections don't have this.)

Anyway, it probably doesn't hurt to leave this.

Reply:
I tested after removing but the filling fails to load properly. I will leave it here for now. Might need a deep dive to remove this code.